<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sf::VariableIdPropertyPage</class>
 <widget class="sf::PropertyPage" name="sf::VariableIdPropertyPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>121</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VariableIdPropertyPage</string>
  </property>
  <layout class="QFormLayout" name="formLayout_2">
   <item row="0" column="1">
    <widget class="sf::InformationIdEdit" name="iieId">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="cbReadOnly">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblId">
     <property name="statusTip">
      <string>Id of the variable.</string>
     </property>
     <property name="text">
      <string>Variable &amp;Id</string>
     </property>
     <property name="buddy">
      <cstring>iieId</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblReadOnly">
     <property name="statusTip">
      <string>Control cannot be used to change the value.</string>
     </property>
     <property name="text">
      <string>load &amp;Only</string>
     </property>
     <property name="buddy">
      <cstring>cbReadOnly</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblConverted">
     <property name="statusTip">
      <string>Show the converted unit.</string>
     </property>
     <property name="text">
      <string>&amp;Converted</string>
     </property>
     <property name="buddy">
      <cstring>cbReadOnly</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="cbConverted">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblNameLevel">
     <property name="statusTip">
      <string>Show the converted unit.</string>
     </property>
     <property name="text">
      <string>&amp;Name Level</string>
     </property>
     <property name="buddy">
      <cstring>sbNameLevel</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="sbNameLevel">
     <property name="minimum">
      <number>-10</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>sf::InformationIdEdit</class>
   <extends>QLineEdit</extends>
   <header>gii/qt/InformationIdEdit.h</header>
  </customwidget>
  <customwidget>
   <class>sf::PropertyPage</class>
   <extends>QWidget</extends>
   <header>misc/qt/PropertyPage.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
